flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemCommitter.java

```diff
@@ -58,7 +59,9 @@ class FileSystemCommitter implements Serializable {
     private final boolean isToLocal;
     private final Path tmpPath;
     private final int partitionColumnSize;
+    private final ObjectIdentifier identifier;
     private final LinkedHashMap<String, String> staticPartitions;
+    private final List<PartitionCommitPolicy> policies;
```

Comment:
`PartitionCommitPolicy` is not serializable, but `FileSystemCommitter` implements `Serializable`. I checked that `FileSystemCommitter` is only used as a local variable. Therefore, `FileSystemCommitter` doesn't need to implement the `Serializable` interface.